src/main/java/teammates/common/datatransfer/FeedbackSessionType.java

```diff
@@ -1,8 +1,8 @@
 package teammates.common.datatransfer;
 
 public enum FeedbackSessionType {
-        STANDARD, PRIVATE;
-        // TODO: be a bit more specific here.
-        // e.g., STANDARD_TEAM_PEER_EVAL, FULLY_CUSTOMIZABLE, PRIVATE, INSTRUCTOR_OBSERVATIONS
-        // Otherwise these will have to be renamed when we add more types later.
```

Comment:
revert back the white space changes?

Reply:
The original is 8-space indentation, which I reduced to 4. Hmm...

Reply:
yeah but this class has nothing to do with this PR... (and after all its gonna get deleted in #8518)